outputs.md

```diff
@@ -17,64 +17,101 @@ FHD outputs various data products. We outline and describe these below. <br />
+  * **freq**: float array of frequency values in Hz
+  * **min_cal_baseline**: float of the minimum baseline length included in calibration
+  * **max_cal_baseline**: float of the maximum baseline included in calibration
+  * **n_vis_cal**: long number of visibilities used in calibration
+  * **time_avg**: integer 1/0 value of whether or not visibilities were averaged in time during the X<sup>2</sup> calibration subroutine
+  * **min_sols**: integer minimum number of equations used in the X<sup>2</sup> calibration subroutine per frequency channel
+  * **ref_antenna**: long number index of tile to reference phases from
+  * **ref_antenna_name**: name of the antenna used as the phase reference.
+  * **conv_thresh**: float number of the required convergence threshold
+  * **polyfit**: integer 1/0 value of whether or not polynomials were fit
+  * **amp_degree**: integer number of degree of coefficients used in fitting the amplitude of the cal solutions i.e. 2 = 2<sup>nd</sup> order polynomial
+  * **phase_degree**: integer number of degree of coefficients used in fitting the phase of the cal solutions i.e. 2 = 2<sup>nd</sup> order polynomial
+  * **bandpass**: integer 1/0 value of whether or not a bandpass was fit
+  * **mode_fit**: integer 1/0 value of whether or not cable reflection mode-fitting was done,
+  * **mean_gain**: !Q
+  * **mean_gain_residual**: !Q
+  * **mean_gain_restrict**: !Q
+  * **stddev_gain_residual**: !Q
+  * **cal_origin**: result of `git describe` run on the FHD repo during calibration. Effectively the git hash, does not include the branch name.
+
+Pointers and large arrays in **cal** structure:
+
+  * **cal.bin_offset**: array of values that specifies the index number of the start of the input visibilities given a time step index. This can be used in conjuncture with cal.uu, cal.vv, cal.tile_a, cal.tile_b, and the like to to find the visibilities associated with a specific time step. For example, to find visibility indices associated with the 20<sup>th</sup> time step, cal.bin_offset[19] specifies the start index and cal.bin_offset[20]-1 specifies the end index.
+
+  * **cal.uu/cal.vv**: array of the light travel time in seconds for the U or V coordinate for each visibility. Visibilities are ordered by iterating through tile A for each tile B, and for each time step. This ordering is the same for cal.tile_a and cal.tile_b, and indices from cal.bin_offset can be used.
+
+  * **cal.tile_a/cal.tile_b**: array of the tile index of the visibility, where tile A and tile B refer to the two separate tiles used in creating that visibility. This ordering is the same for cal.uu and cal.vv, and indices from cal.bin_offset can be used.
+
+  * **cal.gain/cal.gain_residual**: a pointer array of dimension N<sub>pol</sub> which points to an complex array of dimension N<sub>freq</sub> by N<sub>tile</sub>. These arrays contain the complex gain or residual complex gains after all fits have been applied.
+
+  * **cal.auto_params**: !Q
```

Comment:
Npol pointer array, referencing 2 x Ntile arrays of the linear fit offset and slope parameters calculated between the amplitudes of the autocorrelations and cross-correlations for each tile.